src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/OCIArtifactFunctionalTests.cs

```diff
@@ -17,10 +17,10 @@ namespace Microsoft.Health.Fhir.TemplateManagement.FunctionalTests
     public class OCIArtifactFunctionalTests : IAsyncLifetime
     {
         private readonly string _containerRegistryServer;
-        private readonly string _baseLayerTemplatePath = "TestData/TarGzFiles/layer1.tar.gz";
-        private readonly string _userLayerTemplatePath = "TestData/TarGzFiles/layer2.tar.gz";
-        private readonly string _emptySequenceNumberLayerPath = "TestData/TarGzFiles/userV1.tar.gz";
-        private readonly string _invalidCompressedImageLayerPath = "TestData/TarGzFiles/invalid1.tar.gz";
+        private readonly string _baseLayerTemplatePath = Path.Join(new[] { "TestData", "TarGzFiles", "layer1.tar.gz" });
+        private readonly string _userLayerTemplatePath = Path.Join(new[] { "TestData", "TarGzFiles", "layer2.tar.gz" });
+        private readonly string _emptySequenceNumberLayerPath = Path.Join(new[] { "TestData", "TarGzFiles", "userV1.tar.gz" });
+        private readonly string _invalidCompressedImageLayerPath = Path.Join(new[] { "TestData", "TarGzFiles", "invalid1.tar.gz" });
```

Comment:
This is fine, but not actually necessary. Forward slashes will automatically be translated to valid paths on a Windows system, so basically, forward slashes are compatible everywhere. No need to do string concatenation for this, unless you're trying to show some kind of specific intent.

Reply:
Is that true for oras? these are effectively passed to oras directly (i.e. on line 60 of the same file)

Reply:
Definitely. Think about it - it's been running in Windows this whole time using the forward slashes. :)